snap/snapcraft.yaml

```diff
@@ -39,11 +41,15 @@ parts:
       snapdaemon.sh: usr/bin/snapdaemon
 
 plugs:
-   etc-auto-cpufreq-conf:
+  network:
```

Comment:
Why did you add this part? Because now snap fails to build and I can't even upload the new version to beta channel due to:

```
Issues while processing snap:g                                                                                                                                       
- unknown attribute 'default' for interface 'network' (plugs)     
```

Reply:
it was required for giving network/internet access to the env.
it is essential as its a basic necessity for the update to work
though the major issue is due to line 45 i.e. default: true which isn't always  supported 
```     allow-auto-connection: true``` should work
although I'm still figuring out some unexpected errors for lxd
I'm about make a commit, pls check if it still fails
as I was able to build using
```sudo snapcraft --destructive-mode --enable-experimental-extensions```
which builds directly on the host system without using containers which I believe is good for testing